HISTORY.md

```diff
@@ -1,3 +1,183 @@
+## Unreleased
+
+### Changed
+* Remove support for Rubies < 2.1
+* Reduce the number of redis calls when trying to get the list of queues
+* Fix failed jobs page "argument out of range" error
+* Only run eager_load! if Rails.application.config.eager_load is true
+* Don't display log message if running without hooks
+
+## 1.27.4 (2017-04-15)
+
+### Fixed
+* Fix issue where removing a failure from Resque web didn't work when using `RedisMultiQueue` backend.
+
+## 1.27.3 (2017-04-10)
+
+### Fixed
+* Fix issue where initializing a data store would attempt to hit Redis, even when Resque.inline = true
+
+## 1.27.2 (2017-02-20)
+
+### Fixed
+* Require "redis/distributed" in worker.rb to allow proper rescuing
+* Fallback to server time if Redis time won't work (restores Redis 2.4 support)
+
+## 1.27.1 (2017-02-13)
+
+### Fixed
+* Show actual jobs names in web view using ActiveJob (@martnst)
+
+## 1.27.0 (2017-02-08)
+
+### Fixed
+* Fix issue where calling Worker.find, Worker.all, or Worker.working from withing
+  a running job would rewrite the PID file with the PID of the forked worker.
+  This causes a change to the Worker#new API that may affect plugin
+  implementations. See Worker#new and Worker#prepare for details. (@jeremywadsack)
+* Workers queried out now have the correct hostname (@crazymykl)
+* Fix race condition on worker startup (@stevedomin)
+* No longer triggers verbose logging if env variables are not set (@ldnunes)
+* resque/failed/requeue/all when using Redis::Failure::Multiple no longer raises an exception (@ale7714)
+* Improve forking to avoid having a child process escape its code (@dylanahsmith)
+* Workers now use server time rather than their own time to maintain heartbeats (@fw42)
+* Run eager load hooks for Rails applications versioned 4.x and up (@abhi-patel)
+* Fix bug when encountering an error while pruning workers (Joakim Kolsjö and Tomas Skogberg)
+* Children write to PIDFILE immediately after forking, fixing issues when reconnecting to Redis is slow (@fimmtiu)
+
+### Changed
+* Update jQuery from 1.3.2 to 1.12.4 (@chrisccerami)
+* No longer user Thread.kill to stop heartbeat (@Sinjo)
+
+### Added
+* Resque Web UI now prompts for confirmation on clearing failed jobs (Markus Olsen)
+* Adds process status to DirtyExit exception when job is killed via signal (@MishaConway)
+
+## 1.26.0 (2016-03-10)
+
+This changelog is a bit incomplete. We will be much stricter about the changelog for
+the next release.
```

Comment:
Big change: moved a lot of the logic from the Rake task to `Resque::Worker`

https://github.com/resque/resque/commit/7d0b20ee00986b749b03d4407af199039b504cc2